cluster/addons/dashboard/dashboard-controller.yaml

```diff
@@ -56,8 +56,7 @@ spec:
           timeoutSeconds: 30
       volumes:
       - name: kubernetes-dashboard-certs
-        secret:
-          secretName: kubernetes-dashboard-certs
+        emptyDir: {}
```

Comment:
please open an issue for the dashboard to stop trying to write to this location

Reply:
fyi, opened https://github.com/kubernetes/dashboard/pull/2795 to keep the dashboard from writing to this location

Reply:
Would like a dashboard release cut and bumped rather than switching this to emptydir

Reply:
@joelsmith this is still outstanding

Reply:
@liggitt I have opened this issue:
https://github.com/kubernetes/kubernetes/issues/59354
and this PR:
https://github.com/kubernetes/kubernetes/pull/59355
I will contact @floreks about a new release of Dashboard.